addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -391,6 +391,175 @@ options.registry.parallax = options.Class.extend({
     },
 });
 
+options.registry.facebook_page = options.Class.extend({
+    template: 'website.facebook_page_dialog',
+    xmlDependencies: ['/website/static/src/xml/website.facebook_page.xml'],
+    start: function () {
```

Comment:
- Add a line feed between `xmlDependencies` and the first function (here `start`).
- Add a documentation for the `start` method (a `@override` could be enough here).